php/post-types/class-block-post.php

```diff
@@ -61,38 +62,40 @@ public function register_hooks() {
 		add_action( 'wp_ajax_fetch_field_settings', array( $this, 'ajax_field_settings' ) );
 	}
 
-	/**
-	 * Initialise Block posts.
-	 *
-	 * @return void
-	 */
-	public function init() {
-		$this->register_controls();
-	}
-
 	/**
 	 * Register the controls.
 	 *
 	 * @return void
 	 */
 	public function register_controls() {
+		$controls = array(
+			'text'        => new Controls\Text(),
+			'textarea'    => new Controls\Textarea(),
+			'url'         => new Controls\URL(),
+			'email'       => new Controls\Email(),
+			'number'      => new Controls\Number(),
+			'color'       => new Controls\Color(),
+			'image'       => new Controls\Image(),
+			'select'      => new Controls\Select(),
+			'multiselect' => new Controls\Multiselect(),
+			'toggle'      => new Controls\Toggle(),
+			'range'       => new Controls\Range(),
+			'checkbox'    => new Controls\Checkbox(),
+			'radio'       => new Controls\Radio(),
+		);
+
+		if ( block_lab()->is_pro() ) {
+			$controls = array_merge(
+				$controls,
+				array(
+					'user' => new Controls\User(),
+				)
+			);
+		}
+
 		$this->controls = apply_filters(
```

Comment:
Does this fit on one line now?

Reply:
Good point, https://github.com/getblocklab/block-lab/pull/233/commits/f53ba5cf764dadeb84554803502ebc2d94e08a58 moves it to one line.